src/js/device/Device.js

```diff
@@ -388,15 +389,17 @@ export default class Device extends EventEmitter {
         // check if FW version did change
         if (versionCompare(version, this.getVersion()) !== 0) {
             this.unavailableCapabilities = getUnavailableCapabilities(feat, getAllNetworks(), DataManager.getConfig().supportedFirmware);
-            this.firmwareStatus = checkFirmware(version, feat);
-            this.firmwareRelease = getLatestRelease(version);
+            this.firmwareStatus = getFirmwareStatus(feat);
+            // maybe not in bootloade?
+            this.firmwareRelease = getRelease(feat);
         }
         // GetFeatures doesn't return 'session_id'
         if (this.features && this.features.session_id && !feat.session_id) {
             feat.session_id = this.features.session_id;
         }
         this.features = feat;
         this.featuresNeedsReload = false;
+        this.isBitcoinOnly = !!(feat.capabilities && feat.capabilities.length > 0 && !feat.capabilities.includes('Capability_Bitcoin_like'));
```

Comment:
unnecessary `feat.capabilities.length > 0` check

Reply:
removed, will be implemented consumer-side